plugins/readme/src/components/ReadmeCard/ReadmeCard.tsx

```diff
@@ -14,11 +14,12 @@ import { ReadmeDialog } from '../ReadmeDialog/ReadmeDialog';
 
 export type ReadmeCardProps = {
   variant?: InfoCardVariants;
+  maxHeight?: string
```

Comment:
Can we make this type string | number? To make it similar to how Material UI works.